### DISCUSSION TITLE
Uncontrolled Glossary : New strings out of the blue

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#10070] [Q&A]
It happens often that new strings are added to the Glossary out of the blue. It is not clear to me why this strings appear and who added them.

https://translate.codeberg.org/projects/backintime/glossary/#history

I assume there is some kind of automatic mechanic behind it. Am I right?

**Comment by emmapeel2** [comment]
What I know is:

- Translators can be given the power of adding new terms to the glossary.
- The glossary will create a new language for each language you have in any components. So to remove a language from the glossary (maybe a duplicated language, or a fake autogenerated one) you need to remove the language from your other components first.
-  If you mark a term as 'terminology', it will create a new string in each of your languages. If you delete one of them, it will come back. To get rid of it you need to unmark as terminology and then delete all the translations. The source string will then be gone.
